test/uk/gov/hmrc/tai/viewModels/WhatDoYouWantToDoViewModelSpec.scala

```diff
@@ -1,158 +0,0 @@
-/*
- * Copyright 2022 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package uk.gov.hmrc.tai.viewModels
-
-import org.scalatestplus.play.PlaySpec
-import uk.gov.hmrc.tai.model.domain.TaxCodeMismatch
-import uk.gov.hmrc.tai.util.constants.GoogleAnalyticsConstants
-import utils.factories.TaxCodeMismatchFactory
-
-class WhatDoYouWantToDoViewModelSpec extends PlaySpec {
-
-  def gaMap(result: String): Map[String, String] =
-    Map(GoogleAnalyticsConstants.TaiLandingPageInformation -> result)
-
-  def createViewModel(
-    isCyPlusOneEnabled: Boolean,
-    hasTaxCodeChanged: Boolean = false,
-    taxCodeMismatch: Option[TaxCodeMismatch] = None): WhatDoYouWantToDoViewModel =
-    WhatDoYouWantToDoViewModel(isCyPlusOneEnabled, hasTaxCodeChanged, false, taxCodeMismatch)
-
-  val mismatchedTaxCode = TaxCodeMismatchFactory.mismatchedTaxCodeComplex
-  val matchedTaxCode = TaxCodeMismatchFactory.matchedTaxCode
-
-  "showTaxCodeChangeTile" must {
```

Comment:
can we add the showTaxCodeChangeTile tests back in as it looks like it is untested now, unless I have missed them somewhere?

Reply:
no problem

Reply:
added